services/QuillLMS/client/app/bundles/Staff/components/evidence/rulesAnalysis/rulesAnalysis.tsx

```diff
@@ -104,6 +117,12 @@ const RulesAnalysis: React.FC<RouteComponentProps<ActivityRouteProps>> = ({ hist
     queryFn: fetchActivity
   });
 
+  React.useEffect(() => {
```

Comment:
I'm using this hook because we want this to happen at component load time, but also react to the async fetchModels request. Please suggest another technique if this is not ideal.